src/garage/torch/algos/_utils.py

```diff
@@ -3,6 +3,7 @@
 import torch.nn.functional as F
 
 
+# pylint: disable=I0011
```

Comment:
please don't disable informational messages, which don't fail the CI anyway.

Reply:
fixed